### PR TITLE
Python wrappers for temporal/spatial prediction

### PR DESCRIPTION
**Pull request by chuckcho**

### CODE REVIEW

File: action_python/VideoTemporalPrediction.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import math
+import cv2
+import scipy.io as sio
+
+def VideoTemporalPrediction(vid_name, mean_file, net, start_frame=0, num_frames=0):
+
+    num = 25
+    L = 10
+    num_categories = 131
+    feature_layer = 'fc8-2'
```
Comment (yjxiong) [inline action_python/VideoTemporalPrediction.py]: Same as the previous, these may be keyword parameters.

**Comment by yjxiong** [comment]
Thanks a lot! 

I made some minor comments on the codes. Please have a look.

BTW, can this Python wrapper achieve the reported performance?

File: action_python/VideoSpatialPrediction.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import numpy as np
+import math
+import cv2
+import scipy.io as sio
+
+def VideoSpatialPrediction(vid_name, mean_file, net, start_frame=0, num_frames=0):
+
+    num = 25
+    num_categories = 131
+    feature_layer = 'fc8-2'
```
Comment (yjxiong) [inline action_python/VideoSpatialPrediction.py]: These can be keyword parameters to allow easy customization.

**Comment by chuckcho** [comment]
@yjxiong Thanks. I addressed these.  I haven't tested the performance with your reported numbers, but at least I made sure python version results match Matlab-produced ones.

**Comment by yjxiong** [comment]
Merged with thanks!

**Pull request merged**